test/factories/filter_factory.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :filter, :class => ::Filter do
+  end
```

Comment:
Is this block missing a body?

Reply:
I want the factory to exist, but I don't set up any properties. I left the block to be more consistent with other definitions.


I can remove the empty block if it looks annoying for you.

Reply:
It just felt forgotten. I'm ok with it if it is intentional